### ISSUE TITLE
Can we add a note to mention ONIX technique is applicable to UNIMARC?

### ISSUE THREAD

**Issue by gautierchomel** [w3c/publ-a11y#437]
[UNIMARC 231i (pdf file)](https://cdn.ifla.org/wp-content/uploads/U_B_231_update2021_ONLINE_FINAL-1.pdf) offers a easy mapping to ONIX. So the ONIX techniques can be applied to UNIMARC in a predictable formulae.

Example : 
ONIX 196 code 36 will be expressed in UNIMARC as `231 ##$i36$2onix196` (exact syntax to be confirmed).

Should we add this information to Display Guide section 2.3 [Metadata techniques](https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/principles/?updated#techniques)?

**Comment by chrcarr** [comment]
The next meeting of the IFLA Accessibility Metadata Crosswalks Group is December 16.

**Comment by chrcarr** [comment]
@gautierchomel The issue is not with ONIX, but the UNIMARC field, which is only applicable to digital formats. Accessibility details of physical formats cannot be recorded in this field.
